packages/server-core/src/media/file-browser/file-browser.class.ts

```diff
@@ -207,7 +210,12 @@ export class FileBrowserService
     const fileName = await getIncrementalName(data.newName, _newPath, storageProvider, isDirectory)
     const result = await storageProvider.moveObject(data.oldName, fileName, _oldPath, _newPath, data.isCopy)
 
-    await storageProvider.createInvalidation([_oldPath, _newPath])
+    await this.app.service(invalidationPath).create({
```

Comment:
I think this can be done in a single call as

```
await this.app.service(invalidationPath).create([{
  path: _oldPath
},{
  path: _newPath
}])
```